backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -87,4 +81,16 @@ public async Task<List<ProjectResponseDto>> GetAllUserProjectsAsync()
 
         return _mapper.Map<List<ProjectResponseDto>>(userProjects)!;
     }
-}
+
+    private void ValidateProject(Project? entity)
+    {
+        if (entity is null)
+        {
+            throw new EntityNotFoundException(nameof(Project));
+        }
+        if (entity.CreatedBy != _userIdGetter.GetCurrentUserId())
+        {
+            throw new EntityNotFoundException(nameof(Project));
+        }
```

Comment:
you can use a single `if` statement, because the same exception is thrown anyway